http/commands/commands_data.go

```diff
@@ -0,0 +1,77 @@
+package commands
+
+import (
+	"github.com/ravendb-go-client/tools/types"
```

Comment:
These imports are strange. We have all of it in the directory next to it after all..